common/src/logger/mod.rs

```diff
@@ -114,6 +112,13 @@ cfg_if::cfg_if! {
         {
             slog_scope::scope(&logger, || f(&logger))
         }
+    }
+}
+
+cfg_if::cfg_if! {
+    // Time tracing - only available when std is enable, since no_std has no concept of time.
+    if #[cfg(all(feature = "log", feature="std"))] {
```

Comment:
time tracing requires both the logging infrastructure (`Logger`, etc - as provided by `log` as well as the ability to tell the time, which is only available with `std`).